Dockerfile

```diff
@@ -16,19 +17,25 @@ RUN npm ci
 # Application lays in /app/src
 COPY ./auth ./
 
+RUN git clone https://github.com/vishnubob/wait-for-it.git ./scripts
```

Comment:
Why is this needed? Please do not include unchecked code into the image.

Reply:
This "installs" the script "wait-for-it"

Reply:
But you installed it above via apt-get?

Reply:
Also before adding foreign unchecked code, you can do something like this: https://github.com/OpenSlides/openslides-backend/blob/master/dev/wait.sh

Reply:
I could do this in another PR